frame/assets/src/lib.rs

```diff
@@ -1540,10 +1540,15 @@ pub mod pallet {
 			ensure!(origin == details.owner, Error::<T, I>::NoPermission);
 
 			let old_min_balance = details.min_balance;
-			// Ensure that either the new min_balance is less than old min_balance or there aren't
-			// any accounts holding the asset.
+			let is_sufficient = details.is_sufficient;
+			// Ensure that either the new min_balance is less than old
+			// min_balance or there aren't any accounts holding the asset.
+			//
+			// If the asset is marked as sufficient we will only allow the owner
```

Comment:
what is the rationale behind allowing this when there is no accounts holding the asset? I would say that the sufficiency check should apply in any case.

Reply:
You are right, it doesn't make sense.